ci/docker/conda-integration.dockerfile

```diff
@@ -35,7 +35,8 @@ RUN mamba install -q -y \
         maven=${maven} \
         nodejs=${node} \
         yarn \
-        openjdk=${jdk} && \
+        openjdk=${jdk} \
+        zlib=1.2.11 && \
```

Comment:
nit: include the JIRA in a comment so it's clear why we have this pin here?

Reply:
good point! Will do

Reply:
comment is added